src/helpers/add-pr-late-label.ts

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { LATE_REVIEW } from '../constants';
+import { HelperInputs } from '../types/generated';
+import { octokit } from '../octokit';
+import { map } from 'bluebird';
+import { paginateAllOpenPullRequests } from '../utils/paginate-open-pull-requests';
+import { SimplePullRequest } from '../types/github';
+
+export class AddPrLateReviewLabels extends HelperInputs {
+  owner = '';
+  repo = '';
+  days?: string;
+}
+
+export const addPrLateReviewLabels = async ({ owner, repo, days = '1' }: AddPrLateReviewLabels) => {
+  const openPullRequests = await paginateAllOpenPullRequests();
+
+  return map(openPullRequests, pr => {
+    if (!isLabelNeeded(pr, Number(days))) {
+      return;
+    }
+
+    return octokit.issues.addLabels({
+      labels: [LATE_REVIEW],
+      issue_number: pr.id,
```

Comment:
I assume you want the pull request number here, no? I think id is something totally different